buildSrc/src/main/rocker/com/uber/okbuck/template/config/BuckDefs.rocker.raw

```diff
@@ -115,7 +115,7 @@ def okbuck_lint(
     for src in srcs:
         bash += '--sources {}/{} '.format(get_base_path(), src)
 
-    for resource in resources:
+    for resource in res:
```

Comment:
nit: `res_dirs` instead of `res` and `res_dir` instead of `resource`

Reply:
Will fix in follow up

Reply:
Fixed in https://github.com/uber/okbuck/commit/c76e346e29445ce8f3b49aca1ccb6ef1eb41b9e7

Reply:
thanks 👍